modules/network-observability-flowcollector-api-specifications.adoc

```diff
@@ -551,22 +556,22 @@ Required::
 
 | `ipfix`
 | `object`
-| IPFIX configuration, such as the IP address and port to send enriched IPFIX flows to. _Unsupported (*)_.
+| IPFIX configuration, such as the IP address and port to send enriched IPFIX flows to. [Unsupported (*)].
 
 | `kafka`
 | `object`
 | Kafka configuration, such as the address and topic, to send enriched flows to.
 
 | `type`
 | `string`
-| `type` selects the type of exporters. The available options are `KAFKA` and `IPFIX`. `IPFIX` is _unsupported (*)_.
+| `type` selects the type of exporters. The available options are `KAFKA` and `IPFIX`. `IPFIX` is unsupported (*).
```

Comment:
should it be specified as `[Unsupported (*)].` it's inconsistent.

Reply:
@jotak is IPFIX supported for 1.4 release time or is it going out in 4.14 time?

Reply:
Via slack, we discussed IPFIX is supported so this part of the API documentation will need to be updated cc @jotak

Reply:
oops, good catch, I'll update the upstream doc

Reply:
=> https://github.com/netobserv/network-observability-operator/pull/426